### PR TITLE
hotspot_expose() will now heat the turf's air if it fails to create a hotspot

### PR DESCRIPTION
**Pull request by deathride58**
In layman's terms, this means welding tools, bonfires, igniters, etc, will now actually generate heat. Attempting to build a bonfire inside the station without taking precautions is now a very bad idea.

:cl: deathride58
add: Things that are capable of igniting plasma fires will now generate heat if there's no plasma to ignite. Building a bonfire inside the station without taking safety precautions is now a bad idea.
/:cl:

### CODE REVIEW

**Comment by DaxDupont** [comment]
This is a balance change if i've ever seen one

**Comment by deathride58** [comment]
It's actually a surprisingly minor balance change. Doing some testing by spawning 50 welding tools and activating them all, the area I've tested in only got up to 40 degrees celsius after they've run out of fuel. However, bonfires and floor igniters end up heating rooms to beyond lethal temperatures after a little while.

**Comment by Barhandar** [comment]
What happens with cigs?

**Comment by DaxDupont** [comment]
rip bonfire parties.

**Comment by Jalleo** [comment]
Well you can use spaceheaters around the bonfires to keep it cool. Or draught on atmos.

**Comment by ExcessiveUseOfCobblestone** [comment]
THE BOT IS NOT THE END ALL FOR PR LABELING PLEASE STOP COMPLAINING BECAUSE OF THE CL

**Comment by Anonmare** [comment]
BUILD :clap: A :clap: PLASGLASS :clap: WINDOW

**Comment by davethwave** [comment]
Nice PR

**Comment by PKPenguin321** [comment]
Bonfires are unlimited aren't they? Does the heat they produce have a cap? I'm thinking you could make an infinite power generator with a bonfire and a turbine

**Comment by ExcessiveUseOfCobblestone** [comment]
lol

**Comment by kevinz000** [comment]
botany growing power lmao what else is new

**Comment by kevinz000** [comment]
just grow 6 400k cells with botany which will dwarf the infinite power from this.

**Comment by PKPenguin321** [comment]
Also if they're uncapped small rooms would rapidly grow hotter than the sun, etc

**Comment by ike709** [comment]
Make books burn at 451°F.

**Comment by 81Denton** [comment]
It's not as bad as I thought, the temperature has a cap and even five bonfires inside a small room don't instantly bake people.

Floor igniters (/obj/machinery/igniter) only heat up the air on their own tile though, not the whole room.

**Comment by davethwave** [comment]
So basically if you walk onto the tile that's heated up you will be basically be walking into a super pressurized mini area that super hot?

**Comment by optimumtact** [comment]
stale?

**Comment by Michael-Ashfield** [comment]
PLEASE revert, I have just watched greytiders melt the station with igniters and a solo alien on the mother-ship abuse the sparks that generates when he uses the teleport to murder people from safety.

I can understand very small amounts of heat with welders and perhaps lighters, but having assistants being able to make the station an incinerator is over the top.
